dask_ml/model_selection/_hyperband.py

```diff
@@ -0,0 +1,658 @@
+    >>> search.best_params_
+    {'loss': 'log', 'average': False, 'alpha': 0.0080502}
+
+    Attributes
+    ----------
+    cv_results_ : dict of lists
+        Information about the cross validation scores for each model.
+        All lists are ordered the same, and this value can be imported into
+        a pandas DataFrame. This dict has keys of
+
+        * ``rank_test_score``
+        * ``model_id``
+        * ``mean_fit_time``
+        * ``mean_score_time``
+        * ``std_fit_time``
+        * ``std_score_time``
+        * ``mean_test_score``
+        * ``std_test_score``
+        * ``partial_fit_calls``
+        * ``mean_train_score``
+        * ``std_train_score``
+        * ``params``
+        * ``param_value``
+        * ``mean_copy_time``
+
+    meta_ : dict
+        Information about every model that was trained. Can be used as input to
+        :func:`~dask_ml.model_selection.HyperbandCV.fit_metadata`.
+    history_ : list of dicts
+        Information about every model after it is scored. Most models will be
+        in here more than once because poor performing models are "killed" off
+        early.
+    best_params_ : dict
+        The params that produced the best performing model
+    best_estimator_ : any
+        The best performing model
+    best_index_ : int
+        The index of the best performing classifier to be used in
+        ``cv_results_``.
+    n_splits_ : int
+        The number of cross-validation splits.
+    multimetric_ :
+        Whether or whether this model selection algorithm is multimetric.
+    test_size : float, default 0.15
+        The fraction of test set that should be used for testing.
+    best_score : float
+        The best score, updated live as soon as scores are received.
+        This value is available even if
+        :func:`~dask_ml.model_selection.HyperbandCV` does not
+        complete. Note that this value may be higher than ``best_score_``
+        because intermediate values are considered before the model is
+        finished training.
+    best_params : dict
+        The parameters corresponding to ``best_score``, updated live as
+        soon as scores are received.  This value is available even if
+        :func:`~dask_ml.model_selection.HyperbandCV` does not complete.
+
+    Notes
+    -----
+    Hyperband is state of the art via an adaptive scheme. Hyperband
+    only spends time on high-performing models, because our goal is to find
+    the highest performing model. This means that it stops training models
+    that perform poorly.
+
+    The asynchronous variant [2]_ is controlled by the ``asynchronous``
+    keyword and suited for highly parallel architectures. Architectures with
+    little parallism (i.e., few workers) will benefit from
+    ``asynchronous=False``.
+
+    If dask arrays are passed to
+    :func:`~dask_ml.model_selection.HyperbandCV.fit`, the estimator's
+    ``partial_fit`` method is called over each chunk of the array.
```

Comment:
note: called sequentially or in parallel?

Reply:
This is called sequentially.